docs/src/setup/index.md

```diff
@@ -146,4 +146,38 @@ Immediately after logging in with this default user you will be asked to modify
       INITIAL_ADMIN_PASSWORD: mypassword1
 ```
 
+## OpenID Connect - Single Sign-On (SSO)
+
+Nginx Proxy Manager supports single sign-on (SSO) with OpenID Connect. This feature allows you to use an external OpenID Connect provider log in.
+*Note: This feature requires a user to have an existing account to have been created via the "Users" page in the admin interface.*
```

Comment:
```suggestion

::: warning

Please note, that this feature requires a user to have an existing account to have been created via the "Users" page in the admin interface.

:::
```

A small suggestion: include your note in the warning block for consistency with the rest of the document and better visibility on the rendered page.

Reply:
Done!